packages/optimizers/htmlnano/src/HTMLNanoOptimizer.js

```diff
@@ -36,6 +44,10 @@ export default (new Optimizer({
       );
     }
 
+    // $FlowFixMe
+    const presets = htmlnano.presets;
+    const preset =
+      typeof config?.preset === 'string' ? presets[config.preset] : {};
```

Comment:
There should be a test case for using the `preset` config option, given it has special behaviour.